bin/addons-linter.sh

```diff
@@ -9,27 +9,9 @@
 
 . $(dirname $0)/commons.sh
 
-TMPDIR=/tmp/MAC_addonsLinter
-
 print Y "Update the submodules..."
 git submodule init || die
 git submodule update --remote --depth 1 src/_locales || die
 
-printn Y "Removing previous execution data... "
-rm -rf $TMPDIR || die
-print G "done."
-
-printn Y "Creating a tmp folder ($TMPDIR)... "
-mkdir $TMPDIR || die
-print G "done."
-
-printn Y "Copying data... "
-cp -r src $TMPDIR || die
-print G "done."
-
-printn Y "Removing the github folder... "
-rm -rf $TMPDIR/src/_locales/.github || die
-print G "done."
-
 print Y "Running the test..."
-npx addons-linter $TMPDIR/src || die
+npx addons-linter ./src || die
```

Comment:
Nitpicking but I'd prefer if we could run it from any subfolder instead of using `./`.